src/scripts/postclone.js

```diff
@@ -641,8 +641,6 @@ function addPluginToDemoApps() {
                 if (appsToInstallPluginIn.length == 0) {
                     rimraf(screenshots_dir, function () {
                         console.log('Screenshots removed.');
-                        rimraf(seed_tests_dir, function () {
```

Comment:
we remove tests because they test the plugin seed instead of providing a starting point for plugin authors to tests their plugins